packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -617,6 +637,50 @@ describe('Testing authentication capability', () => {
           });
         });
 
+        it(`authentication.getUser should successfully get user profile with data residency info if dataResidency is provided by hosts`, (done) => {
+          utils.initializeWithContext(context).then(() => {
```

Comment:
Is there a way to use async-await instead of callbacks? Is there any particular reason we are using callbacks?
Same goes for all the test cases below.

Reply:
I am following the current `getUser` unit tests format which are using callbacks.